### PR TITLE
Add Dockerfile and initial launch files

### PR DESCRIPTION
**Pull request by GreatAlexander**

### CODE REVIEW

File: av_camera_launch/config/chameleon_config.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/**:
+        ros__parameters:
```
Comment (hect95) [inline av_camera_launch/config/chameleon_config.yaml:2]: This is more style, but, shall we reduce the indentation?

File: av_camera_launch/config/chameleon_config.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                # set parameters defined in chameleon.yaml
+                gain_auto: "Continuous"
+                exposure_auto: "Continuous"
+                offset_x: 0
+                offset_y: 0
+                image_width: 2048
+                image_height: 1536
+                pixel_format: "BayerRG8"  # 'BayerRG8 'RGB8' or 'Mono8'
+                frame_rate_continous: True
+                frame_rate: 5.0
```
Comment (hect95) [inline av_camera_launch/config/chameleon_config.yaml:15]: This is a tricky one, I remember 18-22 fps is a good balance between, resources, vehicle speed and detection for a Full HD camera, what do you think?

**Review by hect95 — COMMENTED** [review]
Minor comments :)

**Comment by hect95** [comment]
@GreatAlexander Docker scripts and file are updated!

File: av_camera_launch/config/chameleon_config.yaml (hunk 2 above)
Comment (GreatAlexander) [reply av_camera_launch/config/chameleon_config.yaml]: So this number is misleading: We cannot control the frame_rate of the camera sensors via this configuration i.e. this number does nothing. I played with it while I was investigating the camera hardware & drivers and came to that conclusion. The camera sensors run at an unlimited / uncapped framerate, which in effect is 10-20Hz (Depending on bandwidth limitations).
Whether it is a bug of the driver or a peculiarity of our hardware I do not know. What I do know is that we are intending to use the camera-trigger hardware/software (Which depends on the GPS PPS signal) to set the framerate and synchronise all camera images.
So what we should do is leave these config values to the default (from what the driver repo config says) and put a comment explaining what I said above (We should do this after testing this in the lab)

Comment (hect95) [reply av_camera_launch/config/chameleon_config.yaml]: Lets do that, we first add whatever conclusion we come up as a comment and merge

Comment (hect95) [reply av_camera_launch/config/chameleon_config.yaml]: Lets do this on different branch as its seems the cameras cfg and ROS drivers needs to be investiaged more. For now we have a working docker!

**Review by hect95 — APPROVED** [review]
Tested on vehicle and ready to go!

**Pull request merged**